metadeploy/api/models.py

```diff
@@ -956,7 +956,15 @@ class ScratchOrg(HashIdMixin, models.Model):
 
     objects = ScratchOrgQuerySet.as_manager()
 
+    def clean_config(self):
+        if self.config:
+            banned_keys = {"email"}
```

Comment:
@davisagli We're still storing `access_token` and `refresh_token`. I don't know if there are security/privacy concerns there.

(I also note on Metecho that we're initially removing `email`, but then it gets re-added at some point in the process of reassigning an org from one user to another -- I didn't dig into it enough to figure out why.)

Reply:
@jgerigmeyer @jerivas The place where we are currently using the access_token and refresh_token is in the run_flows job when it sets up the OrgConfig that will be used to run the flow. When we're running against a scratch org, we should be able to instead use scratch_org.get_refreshed_org_config() to get an org config (including a fresh access token obtained via the JWT oauth flow). Then it would not be necessary to store the access_token and refresh_token in the scratch org model.

However once the flow is running, CumulusCI will try to call org_config.refresh_oauth_token() at various times, and this will fail because there's no refresh_token. Possible solutions to this are:
1. Set the SFDX_CLIENT_ID and SFDX_HUB_KEY environment variables by copying from CONNECTED_APP_CLIENT_ID and CONNECTED_APP_CLIENT_KEY. When SFDX_CLIENT_ID and SFDX_HUB_KEY are set, CumulusCI will also refresh the access token using the JWT oauth flow instead of the refresh token flow.
2. Set org_config.refresh_oauth_token to a function that doesn't do anything. The access token won't get refreshed, but it should last as long as our job timeout anyway.

Reply:
This is something that came up a few weeks ago too, I think it'd be worthwhile to align our env vars with what CCI expects. I have a story on the ice box that we can move up https://trello.com/c/2uQdbFDm/181-as-a-dev-i-want-to-use-the-same-environment-variables-as-cumulusci